src/script/standard.h

```diff
@@ -41,11 +42,11 @@ static const unsigned int MAX_OP_RETURN_RELAY = 83;
 /**
  * A data carrying output is an unspendable output containing data. The script
  * type is designated as TxoutType::NULL_DATA.
+ *
+ * Maximum size of TxoutType::NULL_DATA scripts that this node considers standard.
+ * If nullopt, any size is nonstandard.
```

Comment:
Agree with the approach of optional<size> in fac85fe0632eace9b49d5bdbc0e9e47765b01090 :+1:. At first was unsure about introducing a global `g_max_datacarrier_bytes`, then realized it was moved into policy in fa727f139c38e24fa23a6455b0b274177b41747c.